cc.arduino.arduinoide2.json

```diff
@@ -0,0 +1,119 @@
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+
+    "modules": [
+        "shared-modules/libusb/libusb.json",
+        "python3-pyserial.json",
+        "shared-modules/libsecret/libsecret.json",
+        {
+            "name": "usbutils",
+            "config-opts": [
+                "--datadir=/app/share/hwdata",
+                "--sbindir=/app/bin"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/gregkh/usbutils.git",
+                    "tag": "v014",
+                    "commit": "57fb18e59cce31a50a1ca62d1e192512c905ba00"
+                }
+
+            ]
+        },
+        {
+            "name": "git",
+            "make-args": [
+                "NO_TCLTK=1",
+                "INSTALL_SYMLINKS=1"
+            ],
+            "make-install-args": [
+                "NO_TCLTK=1",
+                "INSTALL_SYMLINKS=1"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.36.1.tar.gz",
+                    "sha256": "37d936fd17c81aa9ddd3dba4e56e88a45fa534ad0ba946454e8ce818760c6a2c"
+                }
+            ]
+        },
+        {
+            "name": "arduino-ide",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mv ./arduino-ide /app/arduino-ide",
+                "install -D arduino-ide.sh /app/bin/arduino-ide",
+                "install -Dm644 cc.arduino.arduinoide2.64.png /app/share/icons/hicolor/64x64/apps/cc.arduino.arduinoide2.png",
+                "install -Dm644 cc.arduino.arduinoide2.128.png /app/share/icons/hicolor/128x128/apps/cc.arduino.arduinoide2.png",
+                "install -Dm644 cc.arduino.arduinoide2.appdata.xml /app/share/appdata/cc.arduino.arduinoide2.appdata.xml",
+                "install -Dm644 cc.arduino.arduinoide2.desktop /app/share/applications/cc.arduino.arduinoide2.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "arduino-ide.sh"
+                },
+                {
+                    "type": "file",
+                    "path": "cc.arduino.arduinoide2.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "cc.arduino.arduinoide2.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "cc.arduino.arduinoide2.64.png"
+                },
+                {
+                    "type": "file",
+                    "path": "cc.arduino.arduinoide2.128.png"
+                },
+                {
+                    "type": "archive",
+                    "dest-filename": "arduino-ide.zip",
+                    "dest": "arduino-ide",
+                    "only-arches": [
+                        "x86_64"
```

Comment:
this is very disappointing to prevent using a RaspberryPi for that.

Reply:
the devs are working on a arm build but it isnt ready yet

Reply:
you could build from source.

Reply:
i have been working this in the background on the x86_64 build but it's taking some work. It is the plan to have arm support but the devs currently list it as a work in progress and i currently dont have any hardware to do any testing yet.
(thread on arm support: https://github.com/arduino/arduino-ide/issues/107)